src/main/scala/nl.knaw.dans.easy.ccw/CommandLineOptions.scala

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.ccw
+
+import org.rogach.scallop.ScallopConf
+
+class CommandLineOptions(args: Array[String], version: String) extends ScallopConf(args) {
```

Comment:
Most projects have a [`ReadmeSpec`](https://github.com/DANS-KNAW/easy-pid-generator/blob/master/src/test/scala/nl/knaw/dans/easy/pid/ReadmeSpec.scala) to check that the command line is in sync with the README. Can you add that one as well?

Reply:
Okay, in this case I didn't get any failure with mvnci related to Readme file

Reply:
that's right, because the `ReadmeSpec` isn't there yet...